deps/sqlite/sqlite3.c

```diff
@@ -24886,8 +24886,8 @@ static const struct {
   /* 1 */ { 6, "minute",   7.7379e+12,        60.0  },
   /* 2 */ { 4, "hour",     1.2897e+11,      3600.0  },
   /* 3 */ { 3, "day",      5373485.0,      86400.0  },
-  /* 4 */ { 5, "month",    176546.0,     2592000.0  },
-  /* 5 */ { 4, "year",     14713.0,     31536000.0  },
+  /* 4 */ { 5, "month",    176546.0,  30.0*86400.0  },
+  /* 5 */ { 4, "year",     14713.0,  365.0*86400.0  },
```

Comment:
Windows build is failing. This needs to be reverted. We can float https://github.com/nodejs/node/commit/15b39026197437493af224f0fe69aa878ec1abd3 here.